plugins/modules/swim_workflow_manager.py

```diff
@@ -1069,14 +1267,23 @@ def get_diff_import(self):
                 self.result['changed'] = False
                 return self
 
+            self.log("image_type - {0}".format(import_type))
             if import_type == "remote":
-                image_name = self.want.get("url_import_details").get("payload")[0].get("source_url")
-            else:
+                image_names = self.want.get("url_import_details").get("payload")[0].get("source_url")
+                self.log(image_names)
+            elif import_type == "local":
                 image_name = self.want.get("local_import_details").get("file_path")
+            else:
+                image_name = self.want.get("cco_import_details").get("image_name")
 
             # Code to check if the image already exists in Catalyst Center
-            name = image_name.split('/')[-1]
-            image_exist = self.is_image_exist(name)
+            if import_type == "remote":
+                for image_name in image_names:
+                    name = image_name.split('/')[-1]
+                    image_exist = self.is_image_exist(name)
```

Comment:
This result will be overwritten everytime and store the result of last image only...how we are going to use this variable - image_exist

Reply:
updated the code